### PR TITLE
Symfony integration

### PR DESCRIPTION
**Pull request by soullivaneuh**
PHP 5.6 is required.

### CODE REVIEW

File: src/Paybox.php [hunk 1]
```diff
@@ -36,13 +36,28 @@
...
+    const VERSIONS = [
+        'direct' => self::VERSION_DIRECT,
+        'direct_plus' => self::VERSION_DIRECT_PLUS,
+    ];
+
     const DEVISE_EURO = 978;
     const DEVISE_US_DOLLAR = 840;
     const DEVISE_CFA = 952;
 
+    const DEVISES = [
```
Comment (greg0ire) [inline src/Paybox.php:51]: `CURRENCIES` ?

File: src/Bundle/DependencyInjection/NexyPayboxDirectExtension.php [hunk 2]
```diff
@@ -60,6 +61,11 @@ private function processOptions(array $config, ContainerBuilder $container)
      */
     private function processClient(array $config, ContainerBuilder $container)
     {
-        // Define http_client_* as private services and set the nexy_paybox_direct.http_client_default one
+        if (null === $config['client']) {
+            return;
+        }
+
+        $httpClientDefinition = $container->findDefinition('nexy_paybox_direct.http_client.'.$config['client']);
```
Comment (greg0ire) [inline src/Bundle/DependencyInjection/NexyPayboxDirectExtension.php]: Tu peux utiliser `getDefinition` direct là je pense

File: src/Paybox.php (hunk 1 above)
Comment (soullivaneuh) [inline src/Paybox.php:51]: Translations will be done in another PR. :+1:

File: src/Bundle/DependencyInjection/NexyPayboxDirectExtension.php (hunk 2 above)
Comment (soullivaneuh) [inline src/Bundle/DependencyInjection/NexyPayboxDirectExtension.php]: Where is your english? :stuck_out_tongue:

Comment (greg0ire) [inline src/Bundle/DependencyInjection/NexyPayboxDirectExtension.php]: Holy shit! Dunno what happened, sorry.
